pages/chaoss/visualizations/contribs_by_action.py

```diff
@@ -0,0 +1,229 @@
+from dash import html, dcc, callback
+import dash
+import dash_bootstrap_components as dbc
+from dash.dependencies import Input, Output, State
+import plotly.graph_objects as go
+import pandas as pd
+import logging
+from dateutil.relativedelta import *  # type: ignore
+import plotly.express as px
+from pages.utils.graph_utils import get_graph_time_values, color_seq
+from queries.contributors_query import contributors_query as ctq
+import io
+from cache_manager.cache_manager import CacheManager as cm
+from pages.utils.job_utils import nodata_graph
+import time
+
+
+PAGE = "chaoss"
+VIZ_ID = "contribs-by-action"
+
+gc_contribs_by_action = dbc.Card(
+    [
+        dbc.CardBody(
+            [
+                html.H3(
+                    "Contributors by Action Type",
+                    className="card-title",
+                    style={"textAlign": "center"},
+                ),
+                dbc.Popover(
+                    [
+                        dbc.PopoverHeader("Graph Info:"),
+                        dbc.PopoverBody(
+                            """Visualizes the number of contributors in the specified time bucket that\n
```

Comment:
I think it's really easy to misunderstand this one and think it's the # of a particular kind of contribution per time range. 

Maybe adjusting to:
"Visualizes the number of contributors who have performed a specific action (have opened a PR, for example) within a specified time-window. This is different from counting the number of contributions (the number of PRs having been opened)- the focus is on the activity of distinct contributors."

The adjustment is minor but I think the clarification is necessary.